### DISCUSSION TITLE
Double click events related to connections

### DISCUSSION THREAD

**Discussion by nanami729** [bcakmakoglu/vue-flow#1209] [Q&A]
May I ask, what should I do if binding a double click event to a straight cannot retrieve its edge?

**Comment by bcakmakoglu** [comment]
I'm not quite following, could you explain in more detail?
Maybe some code, a reproduction, screenshots - anything more than one sentence really 😄

**Reply by nanami729** [reply]
![1702294429755](https://github.com/bcakmakoglu/vue-flow/assets/70675816/c423bd77-86c0-44fb-adac-070afcad3231)
What is the reason for using smoothstep lines, where the middle line is parallel, but the left line is redundant?

**Reply by bcakmakoglu** [reply]
Looks like you just have a hidden node that the edges connect to? 🤔

**Reply by nanami729** [reply]
![1702297395073](https://github.com/bcakmakoglu/vue-flow/assets/70675816/18aa76f3-19ab-447d-b947-921c23ee2645)
The line on the right is protruding, and the middle line will not appear in this state as long as it is not horizontal, without any extra nodes

**Reply by bcakmakoglu** [reply]
Sorry but you completely lost me.
What is the problem? I don't see no "protruding" edges anywhere?

Your post is confusing me in general.
The title says `Double click events related to connections` but none of your posts talk about *any* event.
They all mention some supposed edge misalignment, which I can't even see in your screenshots.

Please elaborate, clearly and in detail *what the issue is* otherwise I can't really help you since I have 0 idea what you're talking about 😞

**Comment by nanami729** [comment]
I'm sorry, my English level is a bit poor, and I apologize for any confusion in my expression. Regarding the title, I have resolved the double click event. The screenshot is another issue, and I cannot see the image, which troubles me. The horizontal connecting line between two blocks will result in a prominent link when using the smoothstep type connection. As long as the blocks are not placed horizontally, there will be no prominent link. I can't think of any reason.
